react/.eslintrc.json

```diff
@@ -1,10 +1,13 @@
 {
   "extends": [
     "../base/.eslintrc.json",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "plugin:react/recommended",
+    "prettier/react"
```

Comment:
wouldn't this be `plugin:prettier/react`?

Reply:
I am not quite sure tbh, copied this from the readme.md

Reply:
as in https://github.com/prettier/eslint-config-prettier